projects/packages/my-jetpack/_inc/components/my-jetpack-screen/styles.module.scss

```diff
@@ -69,6 +70,22 @@
 			align-items: center;
 		}
 	}
+
+	&.is-red-bubble {
+		border-left: 1px solid var(--jp-gray);
+
+		::before {
+			content: '';
+			position: absolute;
+			top: -11px;
+			left: -11px;
```

Comment:
I see where you're going with this and actually -11px is directly centered because of the 2px of padding on either side of the bubble. That makes the bubble 22px total and the middle would be 11px. Though -50% might be better 🤔

Reply:
50% didn't work, with the `absolute` positioning it's going off 50% of the container height, I'll stick with 11px 😄

Reply:
Nice that makes sense - I support that :)